core/src/main/scala/org/scalarelational/instruction/InsertSingle.scala

```diff
@@ -11,8 +11,7 @@ case class InsertSingle(values: Seq[ColumnValue[_]]) extends Insert with Instruc
   override protected def thisDatastore = values.head.column.table.datastore
 
   def result = {
-    val datastore = values.head.column.table.datastore
-    datastore.exec(this)
+    thisDatastore.exec(this)
```

Comment:
I noticed the same in ``Update`` and some other files. Should I submit a small PR?